packages/p2p/src/hooks/useMarketRate.ts

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+import { useExchangeRate } from '@deriv/hooks';
+
+const useMarketRate = () => {
```

Comment:
I suggest creating a new hook called `useP2PExchangeRate` in `@deriv/hooks` to expand the functionality of `useExchangeRate`. This new hook should prioritize the use of the `override_exchange_rate` value while performing all the functions that `useExchangeRate` currently does. By composing different elements, we can create a more comprehensive and effective solution.

Reply:
Farzin I guess we don't need that! we just use this override_exchangerate in 3 of our components that's why I just added a hook for that

Reply:
adding and returning a custom hook in other custom hook dose not make sense :sweat_smile: